.ddev/commands/web/theme

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+## Description: Run yarn inside the default theme directory
+## Usage: theme [flags] [args]
+## Example: "ddev theme install" or "ddev theme watch" or "ddev theme build"
+## ExecRaw: true
+## HostWorkingDir: true
+
+if [ $1 = "watch" ]; then
+    npm --prefix themes/default run watch
+elif [ $1 = "prod" ]; then
+    npm --prefix themes/default run prod
+elif [ $1 = "build" ]; then
+    npm --prefix themes/default run prod
+else
+    npm --prefix themes/default "$@"
```

Comment:
This is mainly to avoid having to either A) move package.json, node_modules etc to the project root, or B) remember the prefix part when running `ddev npm --prefix themes/default install` etc.

So it’d be:

```bash
ddev theme install
ddev theme watch
ddev theme build
```

instead of:

```bash
ddev npm --prefix themes/default install
ddev npm --prefix themes/default run watch
ddev npm --prefix themes/default run prod
```